src/main/java/org/mifos/connector/ams/zeebe/ZeebeUtil.java

```diff
@@ -39,4 +54,75 @@ public static <T> T zeebeVariable(Exchange exchange, String name, Class<T> clazz
         }
         return (T) content;
     }
+
+    public static LoanRepaymentDTO setLoanRepaymentBody(Exchange exchange) {
+        LoanRepaymentDTO loanRepaymentDTO=new LoanRepaymentDTO();
+        loanRepaymentDTO.setDateFormat("dd MMMM yyyy");
+        loanRepaymentDTO.setLocale("en");
+        loanRepaymentDTO.setPaymentTypeId(1);
+        loanRepaymentDTO.setTransactionAmount((BigDecimal) exchange.getProperty("amount"));
+        loanRepaymentDTO.setTransactionDate("02 August 2022");
+        return loanRepaymentDTO;
+    }
+
+    public  Exchange setAccountTypeandNumber(Exchange e,String accountNo){
+        String accountTypeIdentifier="";
+        String accountNumber="";
+        int n=accountNo.length();
+        // Separating account id and prefix
+        for(int i=0;i<n;i++){
+            char c=accountNo.charAt(i);
+            if(Character.isDigit(c)){
```

Comment:
This logic may not work always. The account number may not always start with a digit.  Use the startwith logic with the known prefixes( from configuration)